html/elements/meta.json

```diff
@@ -246,6 +246,9 @@
             "__compat": {
               "description": "http-equiv=\"content-type\"",
               "spec_url": "https://html.spec.whatwg.org/multipage/semantics.html#attr-meta-http-equiv-keyword-content-type",
+              "tags": [
+                "web-features:meta"
+              ],
```

Comment:
Is this the requirement for un-commenting these lines below?

https://github.com/web-platform-dx/web-features/blob/6dd6f01ab8b284091059a2ca9a7275943a0891b6/features/meta.yml#L17-L18

Reply:
That's correct, though we're actually clearing that section altogether since the list of BCD keys is synchronized with BCD tags!